lib/presentation/widget/movie_list.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:auto_size_text/auto_size_text.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import '../../core/util/keys.dart';
+import 'show_image.dart';
+
+import '../../core/util/dimensions.dart';
+import '../../core/util/palette.dart';
+import '../../domain/entity/movie_event.dart';
+
+ListView buildMovieList(MovieEvent movieEvent) {
```

Comment:
You can use a list instead to a MovieEvent

Reply:
Changed it to take a list